plugins/helphub-post-types/classes/class-helphub-post-types-post-type.php

```diff
@@ -491,48 +487,46 @@ public function meta_box_save( $post_id ) {
 
 			switch ( $field_data[ $f ]['type'] ) {
 				case 'url':
-					${$f} = isset( $_POST[ $f ] ) ? esc_url( $_POST[ $f ] ) : '';
+					${$f} = isset( $_POST[ $f ] ) ? esc_url( $_POST[ $f ] ) : ''; /* @codingStandardsIgnoreLine */
 					break;
 				case 'textarea':
 				case 'editor':
-					${$f} = isset( $_POST[ $f ] ) ? wp_kses_post( trim( $_POST[ $f ] ) ) : '';
+					${$f} = isset( $_POST[ $f ] ) ? wp_kses_post( trim( $_POST[ $f ] ) ) : ''; /* @codingStandardsIgnoreLine */
 					break;
 				case 'checkbox':
 					${$f} = isset( $_POST[ $f ] ) ? 'yes' : 'no';
 					break;
 				case 'multicheck':
-					// Ensure checkbox is array and whitelist accepted values against options.
-					${$f} = isset( $_POST[ $f ] ) && is_array( $field_data[ $f ]['options'] ) ? (array) array_intersect( (array) $_POST[ $f ], array_flip( $field_data[ $f ]['options'] ) ) : '';
+					// ensure checkbox is array and whitelist accepted values against options.
+					${$f} = isset( $_POST[ $f ] ) && is_array( $field_data[ $f ]['options'] ) ? (array) array_intersect( (array) $_POST[ $f ], array_flip( $field_data[ $f ]['options'] ) ) : ''; /* @codingStandardsIgnoreLine */
 					break;
 				case 'radio':
 				case 'select':
-					// Whitelist accepted value against options.
+					// whitelist accepted value against options.
 					$values = array();
 					if ( is_array( $field_data[ $f ]['options'] ) ) {
 						$values = array_keys( $field_data[ $f ]['options'] );
 					}
-					${$f} = isset( $_POST[ $f ] ) && in_array( $_POST[ $f ], $values ) ? $_POST[ $f ] : '';
+					${$f} = isset( $_POST[ $f ] ) && in_array( $_POST[ $f ], $values ) ? $_POST[ $f ] : ''; /* @codingStandardsIgnoreLine */
 					break;
 				case 'date':
-					${$f} = isset( $_POST[ $f ] ) ? strtotime( wp_strip_all_tags( $_POST[ $f ] ) ) : '';
+					${$f} = isset( $_POST[ $f ] ) ? strtotime( wp_strip_all_tags( $_POST[ $f ] ) ) : ''; /* @codingStandardsIgnoreLine */
 					break;
 				default :
-					${$f} = isset( $_POST[ $f ] ) ? strip_tags( trim( $_POST[ $f ] ) ) : '';
+					${$f} = isset( $_POST[ $f ] ) ? strip_tags( trim( $_POST[ $f ] ) ) : ''; /* @codingStandardsIgnoreLine */
 					break;
 			}
 
-			// Save it.
-			if ( $f !== 'read_time' ) :
+			// save it.
```

Comment:
Single line comments should begin with a capital letter

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-1-single-line-comments